amivapi/studydocs/endpoints.py

```diff
@@ -80,6 +81,11 @@
                 'maxlength': 30,
                 'nullable': True,
                 'type': 'string'
+            },
+            'coursesemester': {
+                    'maxlength': 5,
+                    'nullable': True,
+                    'type': 'string'
```

Comment:
I am not sure if we gain anything by this. All items already have a creation date - so for proximity you could just sort by that.

Am I missing something?